spec/controllers/home_controller_spec.rb

```diff
@@ -28,9 +28,8 @@
 
       context 'as an admin' do
         before(:each) do
-          DwpCheck.delete_all
           Office.delete_all
-          FactoryGirl.create_list :dwp_check, 2, created_by: manager, office: manager.office
+          FactoryGirl.create_list :benefit_check, 2, user_id: manager.id
```

Comment:
We're usually skipping `FactoryGirl` namespace here as the helpers are included in the specs. Like bellow.